Content/default/README.md

```diff
@@ -1,14 +1,16 @@
 # SAFE Template
+
 This template can be used to generate a full-stack web application using the [SAFE Stack](https://safe-stack.github.io/). It was created using the dotnet [SAFE Template](https://safe-stack.github.io/docs/template-overview/). If you want to learn more about the template why not start with the [quick start](https://safe-stack.github.io/docs/quickstart/) guide?
 
 ## Install pre-requisites
+
 You'll need to install the following pre-requisites in order to build SAFE applications
 
-* The [.NET Core SDK](https://www.microsoft.com/net/download) 3.1 or higher.
-* [npm](https://nodejs.org/en/download/) package manager.
-* [Node LTS](https://nodejs.org/en/download/).
+* [.NET Core SDK](https://www.microsoft.com/net/download) 5.0 or higher
+* [Node LTS](https://nodejs.org/en/download/)
 
 ## Starting the application
+
 Before you run the project **for the first time only** you must install dotnet "local tools" with this command:
```

Comment:
Now that we don't use a dotnet tool to run the build, we can remove this instruction and put `dotnet tool restore` in the FAKE code instead. Another advantage of this approach.

Reply:
I think it's better to continue asking to do `dotnet tool restore` - otherwise you might not have paket installed and restore can fail